miden-tx/tests/integration/wallet/mod.rs

```diff
@@ -62,15 +62,17 @@ fn prove_receive_asset_via_wallet() {
 
     // CONSTRUCT AND EXECUTE TX (Success)
     // --------------------------------------------------------------------------------------------
-    let data_store = MockDataStore::with_existing(Some(target_account.clone()), Some(vec![note]));
+    let data_store = TransactionContextBuilder::new(target_account.clone())
```

Comment:
nit: the variable name should probably be `tx_context`.

Reply:
done